cypress/integration/bulk-edit/in-app/bulk-edit-in-app-former-ids.spec.js

```diff
@@ -0,0 +1,72 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import testTypes from '../../../support/dictionary/testTypes';
+import permissions from '../../../support/dictionary/permissions';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import devTeams from '../../../support/dictionary/devTeams';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import FileManager from '../../../support/utils/fileManager';
+import Users from '../../../support/fragments/users/users';
+import generateItemBarcode from '../../../support/utils/generateItemBarcode';
+import ItemRecordView from '../../../support/fragments/inventory/itemRecordView';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+
+let user;
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: generateItemBarcode(),
+};
+const formerId = generateItemBarcode();
+const validItemFormerIdsFileName = `validItemFormerIds_${getRandomPostfix()}.csv`;
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
```

Comment:
Why don't you use just one string in one describe?

Reply:
answer is here https://github.com/folio-org/stripes-testing/pull/541#discussion_r940135013